### DISCUSSION TITLE
graphql-kotlin-server with Spring MVC

### DISCUSSION THREAD

**Discussion by maespinoza** [ExpediaGroup/graphql-kotlin#1200] [Q&A] [answered]
Hi, is there any plans to provide an example implementation using graphql-kotlin-server with regular Spring MVC servlet-thread-pool based implementation? Some organizations might not be ready to use WebFlux but might be interested to create GraphQL servers.

**Comment by smyrick** [comment, marked as answer]
The maintainers of this library do not have any plans on providing an included implementation that uses Spring MVC, however with the changes we made in version 4, we now have a generic server interface that can be used for any server library, like Ktor.

If you happen to write the code needed for a Spring WebMVC library we would be happy to link to it from our docs. Much of the code is very similar to what we have in `graphql-kotlin-spring-server`, you will just need to replace all the calls with the blocking equivalents
